### DISCUSSION TITLE
Exiting boot services iPad air 2

### DISCUSSION THREAD

**Discussion by Kooza25** [utmapp/UTM#6227] [Linux]
* UTM Version: 4.5.0 (trollstore)
* OS Version: 15.8.2
* Device Model: iPad air 2
* Is it jailbroken (name jailbreak used)? Meowbrek2
* How did you install UTM? Trollstore

Using ubuntu 22.04 pre-made from the library. At first boot it gets stuck there immediately and still does it if quit and retry.

**Comment by osy** [comment]
I think you just have to wait. You are emulating an entire computer. It may take 20-30 minutes.
